modules/ROOT/pages/mongodb/mongodb-connector-6-0-studio.adoc

```diff
@@ -0,0 +1,100 @@
+= MongoDB Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+To configure a connector in Anypoint Studio follow this process:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+== Install the Connector Using Exchange
+
+. In Studio, create a Mule project.
+. Click the Exchange *(X)* icon in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `MongoDB`.
+. Select the MongoDB Connector and click *Add to project*.
+. Follow the prompts to install the connector.
+
+== Install the Connector in Studio
+
+. In Studio, create a Mule project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the name of the connector in the search field.
+. Click the MongoDB Connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source for the Connector
+
+Configure an input source for the connector such as the Object Listener operation, HTTP Listener, or Scheduler. The Object Listener operation invoke your application when MongoDB has a new object in a specified collection.
+
+To configure an input source, drag it to the *Source* area of the Studio canvas.
+
+== Configure the Connector
+
+. On the *Global Elements* tab, click *Create*.
+. Expand *Connector Configuration* and search for `MongoDB config`.
+. Click *OK*.
+. Enter a value for the following properties:
+* *Servers (host:port)*
++
+Comma-separated list of host:port instances. The default value is `localhost:27017`.
++
+* *Database*
++
+Database that sends and retrieves data from the MongoDB instance
++
+* *Username*
++
+Username to access the MongoDB instance
++
+* *Password*
++
+Corresponding password
++
+. In the *Connection* section, select the *Advanced* tab to adjust the values for the *Connection timeout*, *Socket timeout*, and *Maximum connection idle time* field.
```

Comment:
I think we can remove timeout section, it's not required for this example and adds complexity